guides/common/modules/proc_synchronizing-python-repositories.adoc

```diff
@@ -13,17 +13,16 @@ You can view the Python packages in the {ProjectWebUI} at *Content* > *Content T
 +
 The *Label* field is populated automatically based on the name.
 . From the *Type* list, select *python*.
-. In the *Upstream URL* field, enter the URL for the upstream content source, for example `\https://pypi.org`.
+. In the *Upstream URL* field, enter the URL for the upstream content source, for example `https://pypi.org`.
```

Comment:
```suggestion
. In the *Upstream URL* field, enter the URL for the upstream content source, for example, `https://pypi.org`.
```

Reply:
To be honest, sometimes I think we might be overdocumenting the UI. :-)  I assume most users can figure out what "Upstream URL" means.